src/mavedb/view_models/experiment.py

```diff
@@ -72,6 +73,30 @@ def validate_keywords(cls, v):
         keywords.validate_keywords(v)
         return v
 
+    @validator("title")
+    def validate_title(cls, v):
+        if is_null(v) or not isinstance(v, str):
+            raise ValidationError("Invalid title. Title should not be None or space.")
+        return v
+
+    @validator("short_description")
+    def validate_short_description(cls, v):
+        if is_null(v) or not isinstance(v, str):
+            raise ValidationError("Invalid short description. Short description should not be None or space.")
+        return v
+
+    @validator("abstract_text")
+    def validate_abstract_text(cls, v):
+        if is_null(v) or not isinstance(v, str):
+            raise ValidationError("Invalid abstract text. Abstract text should not be None or space.")
+        return v
+
+    @validator("method_text")
+    def validate_method_text(cls, v):
+        if is_null(v) or not isinstance(v, str):
+            raise ValidationError("Invalid method text. Method text should not be None or space.")
+        return v
+
```

Comment:
```suggestion
    @validator("title", "short_description", "abstract_text", "method_text")
    def validate_field_is_non_empty(cls, v, field):
        if is_null(v) or not isinstance(v, str):
            raise ValidationError("This field is required and cannot be empty.")
        return v.strip()
```

Reply:
Thanks Ben! I thought `is_null` function removes the space but in fact, it doesn't. I'll modify the codes.